lib/vfscore/automount.c

```diff
@@ -440,8 +502,8 @@ static int vfscore_extract_volume(const struct vfscore_volume *vv)
 	if (!strcmp(vv->sdev, LIBVFSCORE_EXTRACT_DEV_INITRD0)) {
 		struct ukplat_memregion_desc *initrd;
 
-		rc = ukplat_memregion_find_initrd0(&initrd);
-		if (unlikely(rc < 0 || initrd->len == 0)) {
+		rc = vfscore_get_initrd0(&initrd);
+		if (unlikely(rc < 0)) {
 			uk_pr_crit("Could not find an initrd!\n");
 			return -1;
```

Comment:
Ditto (I know it is not part of the PR)

Reply:
I see this similar in order to be consistent what the automount function returns (always -1 for failure), even if we would update `vfscore_get_initrd()` return codes.